source/_patterns/templates/typography/vertical-rhythm.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Vertical Rhythm
+---
+## Overview
+Vertical Rhythm offers a vertical flow for pages that have content that live outside of components. When the class .pf-vertical-rhythm is applied all typography level tags will receive a margin. All headings will also be given the appropriate size and weight.
+
+## Usage
+
+| Class | Usage |
+| -- | -- |
+| `.pf-vertical-rhythm` **Applied to:** wrapping `<div>, <section> or <article>`|  **Outcome:** applies top and bottom margins to all headings, paragraphs and lists. **Required:** Yes
```

Comment:
Are these the only 3 tags? Would it ever make sense to put it on other tags like `body`, `aside`, `footer`, etc.?